CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Models/GiniImageDocument.swift

```diff
@@ -86,14 +86,25 @@ final public class GiniImageDocument: NSObject, GiniCaptureDocument {
         guard let cgImage = image.cgImage else { return image }
         var updatedRect: CGRect
 
-        print("X: image.size.width \(image.size.width) image.size.height \(image.size.height)")
-
-
         if image.size.width < image.size.height {
-//            updatedRect = CGRect(x: 500, y: 20, width: 2000, height: 2800)
-            updatedRect = CGRect(x: 20, y: 500, width: 2800, height: 2000)
+            if UIDevice.current.isIpad {
+                updatedRect = CGRect(x: 350, y: 350, width: 2400, height: 3300)
```

Comment:
[Clarification] @vizaknaidavid could you please add the details of why you use this specific Rect?

Reply:
This is an older version of the code, there are no more exact numbers there